R/RcppExports.R

```diff
@@ -1187,25 +1199,37 @@ PCO2_LL <- function() {
     .Call('_hector_PCO2_LL', PACKAGE = 'hector')
 }
 
+#' @describeIn ocean Ocean surface partial pressure of CO2
+#' @export
+PCO2 <- function() {
+    .Call('_hector_PCO2', PACKAGE = 'hector')
+}
+
 #' @describeIn ocean Dissolved inorganic carbon, high-latitude
 #' @export
 DIC_HL <- function() {
     .Call('_hector_DIC_HL', PACKAGE = 'hector')
 }
 
+#' @describeIn ocean Ocean surface dissolved inorganic carbon
+#' @export
+DIC <- function() {
+    .Call('_hector_DIC', PACKAGE = 'hector')
+}
+
 #' @describeIn ocean Dissolved inorganic carbon, low-latitude
 #' @export
 DIC_LL <- function() {
     .Call('_hector_DIC_LL', PACKAGE = 'hector')
 }
 
-#' @describeIn ocean Ocean temperature, high-latitude
+#' @describeIn ocean Absolute ocean surface temperature, high-latitude (deg C)
```

Comment:
Following up on our conversation the other day, is this (absolute) correct here?

Reply:
Yup!